src/features/Character/components/Attribute/index.js

```diff
@@ -3,36 +3,37 @@ import styles from './styles.less';
 import TooltipTrigger from 'common/components/TooltipTrigger';
 import classnames from 'classnames/bind';
 import Icon from 'common/components/Icon';
+import T from 'i18n-react';
 
 const cx = classnames.bind(styles);
 
 const attributeNameMapping = {
-  power: 'Attack Power',
```

Comment:
In other translations of gw2 is Power known as what you've called it in German, for example?